### PR TITLE
Tidy _SettingsTab.scss

### PR DESCRIPTION
**Pull request by luixxiul**
For normalization of the tabs on the settings window, removing redundant style rules and making the cascading effect understandable easily.

|Before|After|
|---------|------|
|before|after|

They should be identical.

Signed-off-by: Suguru Hirahara <luixxiul@users.noreply.github.com>

type: task

---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

### CODE REVIEW

File: res/css/views/settings/tabs/_SettingsTab.scss [hunk 1]
```diff
@@ -26,21 +26,19 @@ limitations under the License.
     font-size: $font-20px;
     font-weight: 600;
     color: $primary-content;
-    margin-bottom: 10px;
-    margin-top: 10px;
-}
+    margin-block: 10px; // TODO: Use a spacing variable
```
Comment (weeman1337) [inline res/css/views/settings/tabs/_SettingsTab.scss:29]: I think we should stick here to `margin-bottom`/`margin-top`. If you want to use a shorthand property you can use `margin`. `margin-block` depends on some other attributes. Here the margins are used to define the layout which requires space above and below the element. `margin-block` [can be affected by other CSS properties](https://developer.mozilla.org/en-US/docs/Web/CSS/margin-block).

Applies to other places here as well.

**Review by weeman1337 — CHANGES_REQUESTED** [review]
Hi @luixxiul thank you for your contribution. I've checked the Netlify deployment. LGTM.

Can you please have a look at my comment? I think we should stick to explicit margins if the layout depends on it. If there is something we need to align depending on LTR/RTL we can use the `...start`/`...end` CSS properties.

File: res/css/views/settings/tabs/_SettingsTab.scss (hunk 1 above)
Comment (luixxiul) [reply res/css/views/settings/tabs/_SettingsTab.scss:29]: ok, not really a must unless we would enable UI in [vertical texts](https://www.w3.org/International/articles/vertical-text/) :-D

File: res/css/views/settings/tabs/_SettingsTab.scss [hunk 2]
```diff
@@ -26,80 +30,69 @@ limitations under the License.
...
+    margin-bottom: 10px; // TODO: Use a spacing variable
 }
 
 .mx_SettingsTab_subsectionText {
     color: $settings-subsection-fg-color;
     font-size: $font-14px;
     display: block;
-    margin: 10px 80px 10px 0; // Align with the rest of the view
+    margin-top: 10px; // TODO: Use a spacing variable
+    margin-inline-end: 80px; // Align with the rest of the view
```
Comment (luixxiul) [inline res/css/views/settings/tabs/_SettingsTab.scss:55]: This 80px probably should also be a variable to sync with `$end-gutter` below.

**Review by weeman1337 — APPROVED** [review]
LGTM now. Thanks :+1:

**Pull request merged**